pennylane/_device.py

```diff
@@ -413,7 +438,9 @@ def supports_operation(self, operation):
             if operation.endswith(Operation.string_for_inverse):
                 return operation[
                     : -len(Operation.string_for_inverse)
-                ] in self.operations and self.capabilities().get("inverse_operations", False)
+                ] in self.operations and self.capabilities().get(
+                    "supports_inverse_operations", False
+                )
```

Comment:
Woah, black left this looking really weird

Reply:
Slight tangent, but I've always thought the way we do inverse operations is a bit weird. Really, there are only two gates that need the plugin to implement inverse; S and T. Every single other gate is either its own inverse (PauliZ, Hadamard), or the inverse is a function of the arguments (RX(-x), QubitUnitary(U.conj().T)).

I've always wondered if we could treat inverses like decompositions:

* PL-Core asks the device to implement an inverse of a gate.

* If the device doesn't support the inverse natively, PL fallsback to the above parameter transform, or if unknown, simply raises an error.

Reply:
Having said that, I don't think it's priority to fix, but would be a nice to have.

Reply:
Nice idea indeed!